.travis.yml

```diff
@@ -66,8 +67,34 @@ jobs:
       before_install:
         - ./scripts/if_changed.sh ./scripts/install_prereqs.sh
       script:
+        # Eliminate one warning from BoringSSL when CocoaPods 1.6.0 is available.
+        # The travis_wait is necessary because the command takes more than 10 minutes.
         - travis_wait ./scripts/if_changed.sh bundle exec pod lib lint FirebaseFirestore.podspec --allow-warnings --no-subspecs
 
+    # pod lib lint to check build and warnings for static library build
```

Comment:
I'm not really opposed to this but I'm starting to wonder if we're getting past the point of diminishing returns on pre-commit validation.

Even with the parallelism we now have, builds are taking two and a half hours: https://travis-ci.org/firebase/firebase-ios-sdk/builds. Nothing is taking outrageously long, but we're doing a ton of work now: https://travis-ci.org/firebase/firebase-ios-sdk/builds/377126957

I'm starting to think we should push secondary validation into a [cron job](https://docs.travis-ci.com/user/cron-jobs/) that runs daily.